src/options/options.js

```diff
@@ -31,6 +33,24 @@ Array.prototype.forEach.call(elements.licenseInputs, (element) => {
   });
 });
 
+function initEnableSearchStorageButton() {
+  chrome.storage.sync.get(['enableSearchStorage'], (res) => {
+    enableSearchStorageOption = res.enableSearchStorage;
+    elements.enableSearchStorageCheckbox.checked = enableSearchStorageOption;
+  });
+}
+initEnableSearchStorageButton();
+
+elements.enableSearchStorageCheckbox.addEventListener(('click'), () => {
+  // eslint-disable-next-line quote-props
```

Comment:
We should disable the eslint errors when absolutely necessary. `quote-props` error can easily be handled here. You can remove the quotes around `enableSearchStorage` because as it is not a variable holding any other value, it will be implicitly converted to a string and made a key value in the storage. 
Make sure to check that no other linting error occurs.